src/components/login/login.jsx

```diff
@@ -1,47 +1,71 @@
-import React from "react";
+import React, { useReducer, useState } from "react";
 import './index.css';
-import { Link, useNavigate} from 'react-router-dom';
+import { Link, useNavigate,ReactDOM} from 'react-router-dom';
 import logo from '../../assets/Images/logo.png';
 
-function LoginModule({setToken}){
-    const navigate = useNavigate();
+function LoginModule(){
+    /// Success log in
+    const [loginRes, setLoginRes] = useState();
 
-    const onSubmitHandler = (e) => {
-        e.preventDefault();
-        setToken('123');
-        navigate('/');
-    }
+    let navigate = useNavigate();
 
-    return(
-        <div className="justify-content-center">
-            <div className="icon">
-                <img src={logo} alt="logo"></img>
+    const handleSubmit = (event) => {
+      //Prevent page reload
+      event.preventDefault();
+      
+      var { uname, pass } = document.forms[0];
+      
+      props.login(uname, pass);
+      setLoginRes(props.yourStateName)
+      
+      // if(loginRes is showing login is success) {
+      //   store data (token and userId) to session storage --> check alesandro's code or ask alesandro to work on this
+      //   setIsSubmitted(true);
+      //   navigate('/')
+      // } else { // login failed
+      //   setIsSubmitted(false);
+      // }
```

Comment:
This part can leave out to me, make sure you can fetch the data from the backend first then I will try to store the token.

Simply call console.log(loginRes) on submit button, then make sure you will see the information from the backend

Reply:
Noted